src/fides/api/alembic/migrations/versions/d9237a0c0d5a_add_hidden_column_to_stagedresource.py

```diff
@@ -0,0 +1,28 @@
+"""add `hidden` and `data_use` columns to stagedresource
+
+Revision ID: d9237a0c0d5a
+Revises: c90d46f6d3f2
+Create Date: 2024-11-21 13:18:24.085858
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "d9237a0c0d5a"
+down_revision = "c90d46f6d3f2"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column("stagedresource", sa.Column("hidden", sa.Boolean(), nullable=False))
+    op.add_column("stagedresource", sa.Column("data_use", sa.String(), nullable=True))
```

Comment:
two things:
1. shouldn't this be multi-value/an array? my understanding is we wanted to allow for more than one data use for resource if desired
2. i don't think i see this defined on the SQLAlchemy model in the `models/detection_discovery.py` module?

Reply:
changed to string array and added to model here: https://github.com/ethyca/fides/pull/5600/commits/0832e24a421dded8ff6446991cee0344ac072e44